src/foam/u2/TableView.js

```diff
@@ -46,6 +49,14 @@ foam.CLASS({
   methods: [
     function addObj(obj) {
       var e = this.start('tr')
+          .on('click', function() {
+            this.tableView.selection = obj;
```

Comment:
Is there a way we could attach this listener only once at the table element?  Otherwise we have to make a DOM access for every row which could be slow when scrolling.